main.nf

```diff
@@ -508,9 +509,8 @@ process ericscript {
         -o ./tmp \\
         ${reads}
 
-    if [-f ./tmp/fusions.results.filtered.tsv]; then
-        mv ./tmp/fusions.results.filtered.tsv ./tmp/${sample}_ericscript.tsv
-    fi
```

Comment:
We had this here because the output can be optional.

Reply:
I understood the idea, but it wasn't working for me actually.
I had the `tmp/fusions.results.total.tsv` file in the result directory, but that was all.
The `fusions.results.filtered.tsv` was in the work directory, but not renamed.

I'll try another condition to check the existence of the files then.

Reply:
Done